Sources/FCM/FCMMessage.swift

```diff
@@ -2,18 +2,18 @@ import Foundation
 
 public typealias FCMMessageDefault = FCMMessage<FCMApnsPayload>
 
-public class FCMMessage<APNSPayload>: @unchecked Sendable, Codable where APNSPayload: FCMApnsPayloadProtocol {
+public struct FCMMessage<APNSPayload>: Sendable, Codable where APNSPayload: FCMApnsPayloadProtocol {
```

Comment:
By the way, why are we switching from `class` to `struct`?

Reply:
Because reference types are not inherently thread safe and can result in data races, especially when dealing with mutable shared state (e.g `var`s). Swift 6 requires shared mutable state to be used carefully and therefore complains about classes, and the solution is either making the shared state immutable, or making the type a value type (structure) or preventing concurrent access to the shared state (e.g with a lock). The easiest is simply making the type a structure, as it seems to be we don't need it to be a reference type. If it _does_ need to be a class we can rethink the solution